content/posts/irods-development-update-august-2024.md

```diff
@@ -0,0 +1,260 @@
+Title: iRODS Development Update: August 2024
+Date: 2024-08-14 17:00
+Author: Kory Draughn
+Slug: irods-development-update-august-2024
+Status: published
+
+
+Hello reader,
+
+iRODS 4.3.3 is very close. We expect we'll have a release in a matter of days/weeks.
+
+Starting with the server, it was discovered that the `rc_replica_truncate` API was not returning information in the JSON output parameter as documented. That has been resolved. We've confirmed the server works with MariaDB. For this to work, the MySQL ODBC Connector must be used. Work on the database plugin project template is progressing well. When it is complete, users looking to add support for new databases will be able to jump right into writing code instead of spending time configuring CMake. For users running iRODS on CentOS 7, the Consortium will not be releasing iRODS 4.3.3 packages for this platform.
```

Comment:
```suggestion
Starting with the server, it was discovered that the `rc_replica_truncate` API was not returning information in the JSON output parameter as documented. That has been resolved. We've confirmed the server works with MariaDB. For this to work, the MySQL ODBC Connector must be used. Work on the database plugin project template is progressing well. When it is complete, users looking to add support for new databases will be able to jump right into writing code instead of spending time configuring CMake. The Consortium has also decided not to release iRODS 4.3.3 packages for CentOS 7, as it has reached EOL.
```

Reply:
Done.